minio/api.py

```diff
@@ -296,6 +296,9 @@ def list_buckets(self):
 
         # default for all requests.
         region = 'us-east-1'
+        # region is set then use the region.
```

Comment:
Since we call the same code block twice, we can make it a simple method.

Reply:
There is no reason to make this a method @ebozduman - it will confuse with _get_bucket_location() - i will prefer to keep it as is.